exercises/forth/ForthTest.fs

```diff
@@ -19,12 +19,16 @@ let ``Addition - can add two numbers`` () =
 
 [<Fact(Skip = "Remove to run test")>]
 let ``Addition - errors if there is nothing on the stack`` () =
-    let expected = None
+    let expected = Some {
```

Comment:
You have to update the test generator too, as the expected error comes in a different form. Please see [this PR](https://github.com/exercism/fsharp/pull/613/files#diff-378e45c2fa29f82b16b9d312d5f5c18aR1238) for an example on how to do that.